cluster-install/install.sh

```diff
@@ -1355,11 +1439,14 @@ create_kubectl_patch_for_containers() {
         container_count=1
         logdebug "resource '$resource' in namespace '$namespace' not found, continuing anyway because this is a dry run"
     else
-        fatal "resource '$resource' in namespace '$namespace' was never created"
+        logdebug "resource '$resource' in namespace '$namespace' was never created"
+        _return_error="exceeded timeout waiting for '$resource' in namespace '$namespace' to be created"
+        return 1
```

Comment:
These errors were previously called as `fatal` errors, which wasn't really appropriate because this is just a resource failing to come up, not a flaw in the script logic. "Returning" them will allow the code calling the function to handle the error properly (in this case, adding it as a "problem", with a proper message to the user explaining the failure). I'll point it out in another comment.